doc/libpmem/pmem_flush.3.md

```diff
@@ -157,6 +164,10 @@ it should be used rarely. Typically the application should use this function
 only to flush the most critical data, which are required to recover after
 the power failure.
 
+The behavior of **pmem_deep_drain**() function is the same as **pmem_deep_persist**(),
+except that **pmem_deep_drain**() does not flush processor caches before drain from
+HW buffers and write pending queue flush.
```

Comment:
At least 'before drain from HW buffers and write pending queue flush.' should be removed.

But I think this and previous paragraphs needs to be rewritten. The logical order is: pmem_deep_flush,  pmem_deep_drain and then pmem_deep_persist (as a combination of deep_flush and deep_drain).
Describing deep_persist before deep_drain is confusing...